core/state/intra_block_state_zkevm.go

```diff
@@ -59,7 +59,7 @@ func (sdb *IntraBlockState) GetTxCount() (uint64, error) {
 
 func (sdb *IntraBlockState) PostExecuteStateSet(chainConfig *chain.Config, blockNum uint64, blockInfoRoot *libcommon.Hash) {
 	//ETROG
-	if chainConfig.IsForkID7Etrog(blockNum) {
+	if chainConfig.IsForkID7Etrog(blockNum) && !chainConfig.IsNormalcy(blockNum) {
```

Comment:
For `PostExecuteStateSet`, I think we can simply return immediately if normalcy is enabled.